### ISSUE TITLE
query about output features

### ISSUE THREAD

**Issue by shivansh2524** [wangzheallen/transfer_ijcv#2]
The feature generated using provided script and models are 100-dimensional for each example (in case of chalearn dataset). Can you confirm if these('fc-event' or 'fc' layers) are logits of prediction layer or the some mid level feature layers?
In case these are mid level features, can you share weights of your final prediction layers for replication of results.

Thanks.

**Comment by wangzheallen** [comment]
Hi, thanks for the interest in our work.
can u help verify which the model and prototxt you are using?
